test/utils/Processing.test.ts

```diff
@@ -435,47 +456,47 @@ describe('Processing', () => {
         fhirId: string,
         fhirVersion: string
       ) => {
+        loadedPackages = [];
+        loadedSupplementalFHIRPackages = [];
         const virtualExtensionsConfig = cloneDeep(minimalConfig);
         virtualExtensionsConfig.fhirVersion = [fhirVersion];
         virtualExtensionsConfig.dependencies = [{ packageId: extId, version: fhirVersion }];
         const defs = new FHIRDefinitions();
         return loadExternalDependencies(defs, virtualExtensionsConfig).then(() => {
-          expect(defs.packages.length).toBe(1);
-          expect(defs.packages).toContain(`${fhirId}#${fhirVersion}`);
-          expect(defs.supplementalFHIRPackages).toEqual([`${suppFhirId}#${suppFhirVersion}`]);
+          expect(loadedPackages.length).toBe(1);
+          expect(loadedPackages).toContain(`${fhirId}#${fhirVersion}`);
+          expect(loadedSupplementalFHIRPackages).toEqual([`${suppFhirId}#${suppFhirVersion}`]);
           expect(loggerSpy.getAllLogs('error')).toHaveLength(0);
         });
       };
-      return Promise.all([
-        testExtPackage(
-          'hl7.fhir.extensions.r2',
-          'hl7.fhir.r2.core',
-          '1.0.2',
-          'hl7.fhir.r4.core',
-          '4.0.1'
-        ),
-        testExtPackage(
-          'hl7.fhir.extensions.r3',
-          'hl7.fhir.r3.core',
-          '3.0.2',
-          'hl7.fhir.r4.core',
-          '4.0.1'
-        ),
-        testExtPackage(
-          'hl7.fhir.extensions.r4',
-          'hl7.fhir.r4.core',
-          '4.0.1',
-          'hl7.fhir.r5.core',
-          '4.5.0'
-        ),
-        testExtPackage(
+      await testExtPackage(
+        'hl7.fhir.extensions.r2',
+        'hl7.fhir.r2.core',
+        '1.0.2',
+        'hl7.fhir.r4.core',
+        '4.0.1'
+      );
+      await testExtPackage(
+        'hl7.fhir.extensions.r3',
+        'hl7.fhir.r3.core',
+        '3.0.2',
+        'hl7.fhir.r4.core',
+        '4.0.1'
+      );
+      await testExtPackage(
+        'hl7.fhir.extensions.r4',
+        'hl7.fhir.r4.core',
+        '4.0.1',
+        'hl7.fhir.r5.core',
+        '4.5.0'
+      ),
+        await testExtPackage(
```

Comment:
This looks like it's indented a little extra?

Reply:
I guess prettier let me down. I'll fix!

Reply:
*gasp* I apologize to Prettier for throwing it under the 🚌 . There was a comma instead of a semicolon!

Reply:
Fixed in [377ba4f](https://github.com/FHIR/sushi/pull/1026/commits/377ba4f0f70bbf9b2226773fb692d3662cde7211).